pages/inspector/connect-inspector-to-segment.mdx

```diff
@@ -1,34 +1,18 @@
-# Send data from Segment to Avo Inspector (no code needed)
+# Send data from Segment to Avo Inspector (Native Destination)
 
 ## Use Avo Inspector as a Segment destination
 
 If you are already sending your data to Segment you can stream it to Avo Inspector without adding any code to your codebase.
```

Comment:
```suggestion
Stream your event schemas to Segment to Avo Inspector without adding any code to your codebase.
```

Reply:
this should be "Stream your event schemas from Segment..." right? ill accept this and modify